projects/Group_detr/README.md

```diff
@@ -0,0 +1,33 @@
+# Group DETR
+
+> [FAST DETR TRAINING WITH GROUPWISE ONE-TO-MANY ASSIGNMENT](https://arxiv.org/abs/2207.13085)
+
+<!-- [ALGORITHM] -->
+
+## Abstract
+
+Detection Transformer (DETR) relies on One-to-One assignment, i.e., assigning one ground-truth object to only one positive object query, for end-to-end object detection and lacks the capability of exploiting multiple positive object queries. Group DETR is proposed to support Group-wise One-to-Many assignment. It makes simple modifications during training: (i) adopt K groups of object queries; (ii) conduct decoder self-attention on each group of object queries with the same parameters; (iii) perform One-to-One label assignment for each group, leading to K positive object queries for each ground-truth object. In inference, It only uses one group of object queries, making no modifications to DETR architecture and processes. It is effective on DETR variants, including Conditional DETR, DAB-DETR, DN-DETR, and DINO.
+
+![img.png](img.png)
```

Comment:
we only need to copy these images into a github issue, obtain their link, and paste the link here, so that we do not need to maintain the images.